scripts/wrappers/status.py

```diff
@@ -2,7 +2,7 @@
 import os
 import argparse
 
-from common.utils import (
+from .common.utils import (
```

Comment:
Seems to be necessary for testing, I'm not 100% sure what it does, but it seems to work and break nothing.